pipenv/core.py

```diff
@@ -1529,7 +1526,7 @@ def pip_install(
             )
 
     pip_command = [
-        project._which("python", allow_global=allow_global),
```

Comment:
I wonder how many other issues this method causes... and there is also `project.which`... what a horror ...